pkg/pulumiyaml/analyser.go

```diff
@@ -597,10 +601,12 @@ func (tc *typeCache) typeResource(r *Runner, node resourceNode) bool {
 		)
 	}
 
+	_, skipCheckProps := skipCheckPropsResourceNames[typ.String()]
```

Comment:
keyに対応するvalueが一つ目、keyが存在するかどうかが二つ目に返る（省略可）ので正しくはこんな感じでしょうか？
```suggestion
	skipCheckProps := skipCheckPropsResourceNames[typ.String()]
```